geometry_msgs/PoseStamped pose 

string child_frame_id 

Bearing[] array
